extensions/sources/sql/ext/sql.go

```diff
@@ -28,8 +30,10 @@ import (
 )
 
 type sqlConConfig struct {
-	Interval int    `json:"interval"`
-	Url      string `json:"url"`
+	Interval             int    `json:"interval"`
+	Url                  string `json:"url"`
+	MaxAttempts          int    `json:"maxAttempts"`
+	MaxReconnectInterval int    `json:"maxReconnectInterval"`
```

Comment:
The SQL source is kind of pull source. It will do another pull after an interval which is kind of a retry. So I don't think we need another interval here. Just need a maxAttempts in one time